db_client/data_migrations/taxonomy_cclw.py

```diff
@@ -38,6 +38,12 @@
         "file_key_path": "name",
         "allow_blanks": True,
     },
+    {
```

Comment:
Are you sure this won't affect the older migrations? Even if it does I'm not sure how worried I am yet - but just wondered if you'd considered it?

Reply:
It will affect the older migrations as it's a function in a module that's been updated outside of the alembic data migration file. I can separate it out easy - but also worthy of noting we have done the same thing in other migrations as well (I think this is why the downgrade started failing before the generic corpus work).

Reply:
Ooo have we? Do you have an example?

Reply:
@diversemix have separated, so migration 0033 is not affected. Thinking that if the `get_unf3c_taxonomy` or `get_cclw_taxonomy` functions are ever updated though, this will affect many migrations - is it better to copy the code for these into the alembic migration script to prevent this being an issue, and leaving the helper functions?